account.go

```diff
@@ -252,7 +248,7 @@ type Account struct {
 		UserAgent string `json:"user_agent"`
 	} `json:"tos_acceptance"`
 
-	Type AccountType
+	Type AccountType `json:"type"`
```

Comment:
so it worked without the `json` part though felt inconsistent. Is the library doing some magic and de-serializing if property name matches the JSON?